datalad/local/add_readme.py

```diff
@@ -82,6 +89,23 @@ def __call__(dataset, filename='README.md', existing='skip'):
         # unlock, file could be annexed
         if lexists(filename):
             dataset.unlock(filename)
+        if not lexists(filename):
+            # if it does not exists, make sure it is saved into Git
```

Comment:
Well, now the comment is kinda outdated in this spot. :-P

Reply:
haha, fixed ;-)